composer.json

```diff
@@ -58,5 +57,11 @@
             "id": "01C38Z7F737WGJJFV4MR6GK2SF",
             "allow-contrib": "true"
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "path",
+            "url": "../symfony"
```

Comment:
That might not play well with somebody else checking this repository 😅. I'll update once the AMQP bits are merged.